client/src/components/Settings/OptTextsSwitch.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import firebase from 'firebase';
+
+// Redux imports
+import { connect } from 'react-redux';
+import { getSettings, updateSettings } from '../../actions/settingsActions.js';
+
+// Material UI Components
+import { withStyles } from '@material-ui/core/styles';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Switch from '@material-ui/core/Switch';
+import Typography from "@material-ui/core/Typography";
+
+// WithStyles
+import styles from './styles';
```

Comment:
This will need to be changed to import styles from "../Styles/Settings/styles";
As for me it didnt work with just ./styles

Reply:
Updated! Thanks for finding that.